hrm-domain/lambdas/export-json-writer/src/entityNotification.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+import parseISO from 'date-fns/parseISO';
+import { HrmAccountId } from '@tech-matters/types';
+
+import { Contact, CaseService, ProfileWithRelationships } from '@tech-matters/hrm-types';
+
+type EntityNotificationHeaders = {
+  accountSid: HrmAccountId;
+  operation: 'update' | 'create' | 'reexport';
```

Comment:
I wonder if we could import these from the shared type, so that if changes are made (e.g. a `update` -> `modify`) all dependencies are flagged by TS.